app/controllers/components/export_csv.php

```diff
@@ -127,7 +127,7 @@ function createCsv($params, $event)
      * @access public
      * @return void
      */
-    function generateHeader($params, $event, $peerEval = 1)
+    function generateHeader($params, &$event, $peerEval = 1)
```

Comment:
PHP implements copy-on-write: https://en.wikipedia.org/wiki/Copy-on-write. Unless you are changing the array in place, there will be no benefit using pass-by-reference.  The same applies to foreach loop as well.